code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm

```diff
@@ -565,4 +565,103 @@
 		M.emote(pick("twitch","drool","moan","vomit","flip","scream"))
 	..()
 	return
-*/
\ No newline at end of file
+*/
+
+/datum/reagent/drug/burpium
+	name = "Burpium"
+	id = "burpium"
+	description = "A chemical compound that promotes concentrated production of gas in your esophagus."
+	color = "#c13f9d" // rgb(193, 63, 157)
+	reagent_state = LIQUID
+	overdose_threshold = 40
+
+/datum/reagent/drug/burpium/on_mob_life(mob/living/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(15))
+			H.emote("burp")
+	..()
+	return
+
+/datum/reagent/drug/burpium/overdose_process(mob/living/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(35))
+			H.emote("burp")
+		if(prob(12))
+			if(H.nutrition > 50)
+				H.emote("vomit")
+		if(prob(10))
+			H.setEarDamage(M.ear_damage + rand(0, 5), max(M.ear_deaf,15))
+			if (H.ear_damage >= 15)
+				H << "<span class='warning'>Your ears are ringing badly from all of the burping!</span>"
+				if(prob(H.ear_damage - 10 + 5))
+					H << "<span class='warning'>You can't hear anything!</span>"
+					H.disabilities |= DEAF
+			else
+				if (H.ear_damage >= 5)
+					H << "<span class='warning'>Your ears are ringing from all of the burping!</span>"
+	..()
+	return
+
+/datum/reagent/drug/spookium
+	name = "Spookium"
+	id = "spookium"
+	description = "Just looking at this stuff gives you the chills."
+	color = "#d6d6d6" //rgb(214, 214, 214)
+	reagent_state = LIQUID
+	overdose_threshold = 40
+
+/datum/reagent/drug/spookium/on_mob_life(mob/living/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(7))
+			if(!H.stat == UNCONSCIOUS)
+				H.emote("scream")
+	..()
+	return
+
+/datum/reagent/drug/spookium/overdose_process(mob/living/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(26))
+			if(!H.stat == UNCONSCIOUS)
+				H.emote("scream")
+		if(prob(3))
+			if(!istype(H.dna.species, /datum/species/skeleton))
+				H.visible_message("<span class='danger'>Holy shit! [H] got so scared that their skin tore clean off to reveal a spooky scary skeleton!</span>")
+				H.set_species(/datum/species/skeleton/playable)
+				gibs(H.loc, H.viruses, H.dna)
+	..()
+	return
+
+/datum/reagent/drug/flipvium
+	name = "Flipvium"
+	id = "flipvium"
+	description = "It looks like it wants to do a backflip!"
+	color = "#fcdd80" //rgb(252, 221, 128)
+	reagent_state = LIQUID
+	overdose_threshold = 50
+
+/datum/reagent/drug/flipvium/on_mob_life(mob/living/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(14))
+			if(H.getStaminaLoss() < H.health)
+				H.emote("flip")
+	..()
+	return
+
+/datum/reagent/drug/flipvium/overdose_process(mob/living/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(26))
+			H.emote("flip")
+		if(prob(10))
+			if(H.getStaminaLoss() < H.health)
+				H.adjustStaminaLoss(rand(5,15))
+			if(H.getStaminaLoss() > H.health)
+				H << "<span class='warning'>You feel really tired from all of that flipping!</span>"
+	..()
```

Comment:
It'd be fucking neat if you popped in a prob (5) transform *= -1 in here. SUPERFLIP